python/xorbits/_mars/services/cluster/api/web.py

```diff
@@ -193,6 +193,14 @@ async def fetch_node_log(self):
                     size, address=address, offset=offset
                 )
 
+    @web_api("workers", method=["post"], cache_blocking=True)
```

Comment:
I think the rest api path should describe the behavior of scale up.
If you just set the "workers" here, what api path should be used when implementing the "release_workers" interface in the future?